privacyidea/static/components/audit/views/audit.log.html

```diff
@@ -22,77 +22,91 @@
         <table class="table table-bordered table-striped" id="tableauditlog">
             <thead>
             <tr>
-                <th translate>number</th>
-                <th class="pifilter"><translate>date</translate>
+                <th ng-hide="!(auditdata.auditdata[0].hasOwnProperty('number'))"><translate>number</translate></th>
```

Comment:
Could it ever happen, that we have no entry in the list ``auditdata.auditdata``?
Then we would get a javascript error here.

Another possiblity would be to add the information about the hidden columns as a fixed information here:
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/audit.py#L114
in a static value like

    "hidden_columns": ["number", "sig_data"]

then we could simply check for the existance in the list:

    <th ng-hide=auditdata.hidden_columns.includes("number) translate>number</th>

(Also: You do not need to add an extra tag ``translate``. Simply put it as aparameter in the ``th`` tag.

This also applies to the other occurrences.

Reply:
done in #2500 commit https://github.com/privacyidea/privacyidea/pull/2500/commits/1a81fded8e075b728d2485b0851ce2c553cd3e87